test_scripts/Polices/build_options/032_ATF_P_TC_PTU_Retry_Sequence_Retry_Timeout_Expiration_HTTP.lua

```diff
@@ -148,4 +148,4 @@ function Test.Postcondition_Stop_SDL()
   StopSDL()
 end
 
-return Test
\ No newline at end of file
+return Test
```

Comment:
@istoimenova, Please restore original sdl_preloaded_pt.json file in postconditions